packages/ra-core/src/core/useConfigureAdminRouterFromChildren.tsx

```diff
@@ -43,7 +43,9 @@ import { useResourceDefinitionContext } from './useResourceDefinitionContext';
 export const useConfigureAdminRouterFromChildren = (
     children: AdminChildren
 ): RoutesAndResources & { status: AdminRouterStatus } => {
-    const { permissions, isLoading } = usePermissions();
+    const { permissions, isLoading } = usePermissions(undefined, {
+        enabled: !!getSingleChildFunction(children),
```

Comment:
We should also ensure we don't loose the default query options defined in `usePermissions` (`staleTime`).

Reply:
It would be better to update the `usePermissions` so that it merges the provided options with the default options